apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-modal.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import Search from './location-autocomplete-textbox';
+import { CustomTextInput } from '../../../components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons';
+import { MullBackButton } from '../../../components';
+import { EventRestriction } from '@mull/types';
+import { FormikConfig } from 'formik';
+
+export interface LocationAutocompleteModalProps {
+  formik: FormikConfig<{
+    activeRestriction: EventRestriction;
+    startDate: any;
+    endDate: any;
+    startTime: string;
+    endTime: string;
+    eventTitle: string;
+    description: string;
+    location: string;
+    imageFile: string;
+  }>;
+}
+
+export default function LocationAutocompleteModal({ formik: { touched, setFieldValue, errors } }) {
+  const [inputValue, setInputValue] = React.useState('');
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = (value) => {
+    setOpen(false);
+    if (typeof value == 'string') {
+      setInputValue(value);
+      setFieldValue('location', value);
```

Comment:
Agreed, it should show "Current Location"

Reply:
That's just a place holder, because we might need long and lat in the future for the backend, and this clearly shows the next person how to get it

Reply:
I think my suggestion wasn't clear. I didn't mean that we should change anything about the lat long value, I just want to change what is shown to the user through what shows up in the textbox once they choose to use their current location

Reply:
yeah I got what you mean, but I think displaying lat and long for now is going to be clear for the next person who will use lat and long

Reply:
Do you mean the next people who will be working on this implementation? I don't think this is a supporting ticket for anything

Reply:
Theres #124 which is gonna handle some of the backend since we broke this ticket down into front-end and back-end tasks since this ticket a full user story before we started breaking tickets down

Reply:
True, but this comment is just related to the front-end, and what is shown to the user. In any case I see it was fixed, so I'll be resolving this comment.